modules/33_CDR/DAC/equations.gms

```diff
@@ -33,12 +35,36 @@ q33_emicdrregi(t,regi)..
 	v33_emiDAC(t,regi);
 	
 ***---------------------------------------------------------------------------
-*'  Calculation of energy demand of direct air capture. DAC demands feels and fehes
+*'  Calculation of electricity demand for ventilation of direct air capture.
```

Comment:
I think, the implementation around heat and electricity demand equations can be simplified and written more flexibly. I write you on Mattermost.